packages/mds-compliance/api.ts

```diff
@@ -94,11 +94,11 @@ function api(app: express.Express): express.Express {
     if (!isUUID(policy_uuid)) {
       return res.status(400).send({ err: 'bad_param' })
     }
-    const { start_date, end_date } = query_end_date
-      ? { end_date: query_end_date, start_date: query_end_date - days(365) }
-      : { end_date: now() + days(365), start_date: now() - days(365) }
+
+    const timestamp = query_end_date || now()
```

Comment:
Proposal: Breaking change (but we have nothing querying for historical compliance and the spec is still in PR so should be okay

We should just change the `query_end_date` to a query param of `timestamp`, then we can just do a one-liner and assign a default value on line 81

Reply:
I support renaming, but I'd rather not assign a default value. Easier to condition on `timestamp` being null when determining whether to get historical events or read from the cache that way.